tests/components/plugwise/conftest.py

```diff
@@ -0,0 +1,172 @@
+"""Setup mocks for the Plugwise integration tests."""
+
+from functools import partial
+import os
+import re
+
+from Plugwise_Smile.Smile import Smile
+import jsonpickle
+import pytest
+
+from tests.async_mock import AsyncMock, patch
+from tests.test_util.aiohttp import AiohttpClientMocker
+
+
+def _read_json(environment, call):
+    """Undecode the json data."""
+    fixture = call + ".json"
+    path = os.path.join(
+        os.path.dirname(__file__), "../../fixtures/plugwise/" + environment, fixture
+    )
+    with open(path) as fixture_file:
```

Comment:
Now we can use our fixture helper to read the file.
https://github.com/home-assistant/core/blob/ae5d8f4d64870b522e47319b30b48cc32b2a0ab4/tests/common.py#L327-L331

Reply:
True, started with that but we now know why I deviated :)

Reply:
added (though still as a function def, I can inline it if necessary)

Reply:
Great!